04/User.js

```diff
@@ -1,3 +1,36 @@
 export default class User {
+    constructor({email,password}){
+        this.email = email;
+        this.password = password;
 
+        const regExpEmail = /^[-\w\.]+@([-\w]+\.)+[a-z]+$/i;
+        if(!this.email.match(regExpEmail)){
+            throw new Error('email is incorrect')
+        }
+
+        const regExpPass = /.*?(?:[a-z].*?[0-9]|[0-9].*?[a-z]).*?/
+        if(!this.password.match(regExpPass)){
+            throw new Error('password should has latters and characters')
+        }
+    }
+
+    getEmail(){
+        if(this.email){
```

Comment:
Czy faktycznie musimy sprawdzać? Wtedy i tak zwracamy niejawnie `undefined` bo tak działą funkcja. 
Jak nie ma `return` to zwraca `undefined` :) czyli wychodzi na to samo ;)